### ISSUE TITLE
Editing rpg-skilltrees and rpg-playerstats team's design documents to emphasize difference the two teams

### ISSUE THREAD

**Issue by VannStrasen** [uchicago-cs/chiventure#484]
The border between player stats and skill trees is not well defined in regards to the stats of the skill themself. If a player's skill deals 'x' damage and has a 'y' second cooldown, would that be something that team skill trees or team player stats takes care of? These two teams need to meet up to make sure they both understand who is implementing this specific case, and afterwards, edit their own design documents so that duplicate code isn't produced, or some code isn't made at all.

**Comment by VannStrasen** [comment]
I met with @richardxd today, and we exchanged our design ideas and documented in our own wiki (https://github.com/uchicago-cs/chiventure/wiki/Design-Documentation-for-Skill-Trees-Implementation) about the dependencies regarding our own teams. Specifically, it seems like the example that I gave isn't something that player stats is going to be working with, but from Richard's experience working with battle systems, he says it might be a good idea to talk with them to see how the example I gave above might work and whether that'd be our jobs to implement or battle system's job to implement. Ultimately, player stats will not be working inside of the stats of an active skill, but they will be allowing functionality whereby we could understand how much experience a player is getting and link a level-up system to a skill if needed.
